tools/cppcheck/suppressions-list

```diff
@@ -1,3 +1,4 @@
 operatorEqVarError
 noConstructor
 duplicateExpression
+variableScope
```

Comment:
I checked and you are right. This is definitely a false positive. I actually like the variable scope check, so I'd like to keep it. If there is only one false positive then we can suppress only that with a `//cppcheck-suppress variableScope` comment in the previous line (jerry-core/ecma/builtin-objects/ecma-builtin-helpers.cpp:86). Could you try it?